holoviews/tests/element/test_graphelement.py

```diff
@@ -227,9 +227,9 @@ def test_chord_constructor_with_vdims(self):
 
     def test_chord_constructor_self_reference(self):
         chord = Chord([('A', 'B', 2), ('B', 'A', 3), ('A', 'A', 2)])
-        nodes = np.array(
-            [[-0.5, 0.866025, 0],
-             [0.5, -0.866025, 1]]
+        nodes = pd.DataFrame(
```

Comment:
It could be at some point this worked with an array. This seem like a safe change to made.